configure.ac

```diff
@@ -7,6 +7,9 @@ AH_TOP([#ifndef LIBSECP256K1_CONFIG_H])
 AH_TOP([#define LIBSECP256K1_CONFIG_H])
 AH_BOTTOM([#endif /*LIBSECP256K1_CONFIG_H*/])
 AM_INIT_AUTOMAKE([foreign subdir-objects])
+
+# Set -g if CFLAGS are not already set
```

Comment:
```suggestion
# Set -g but not -O2 if CFLAGS are not already set (see AC_PROG_CC in the Autoconf manual)
```

This is  obscure enough to deserve a comment.

ACK otherwise

Reply:
fixed

Reply:
Eh, anything starting with `AC_` in an error if it occurs in a comment:
```
configure:3073: error: possibly undefined macro: AC_PROG_CC
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
```
Now fixed for real